esileapclient/osc/v1/mdc/mdc_lease.py

```diff
@@ -0,0 +1,105 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import logging
+
+import openstack
+from osc_lib.command import command
+from osc_lib import utils as oscutils
+
+from esileapclient.v1 import client as esileapclient
+from esileapclient.v1.lease import Lease as LEASE_RESOURCE
+
+LOG = logging.getLogger(__name__)
+
+
+class MDCListLease(command.Lister):
+    """List leases across multiple data centers."""
+
+    log = logging.getLogger(__name__ + ".MDCListLease")
+    auth_required = False
+
+    def get_parser(self, prog_name):
+        parser = super(MDCListLease, self).get_parser(prog_name)
+
+        parser.add_argument(
+            '--clouds',
+            dest='clouds',
+            metavar='<clouds>',
+            nargs="+",
+            help="Specify the cloud to use from clouds.yaml."
+        )
+        parser.add_argument(
+            '--long',
```

Comment:
Are we ok with displaying detailed information, when the fields of a lessee could update what the columns are? Two datacenters could be running different versions of esileap. We purposely didn't do this for ironic after all, though Ironic does change faster than esileap, and also we are the ones responsible for changing esileap

Reply:
Good point! I've removed the '--long' option.